src/sentry/models/eventattachment.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import BoundedBigIntegerField, FlexibleForeignKey, Model, sane_repr
+
+
+class EventAttachment(Model):
+    __core__ = False
+
+    project_id = BoundedBigIntegerField()
```

Comment:
this model follows the same principles as UserReport where we expect to be able to handle an attachment uploaded before a Group is associated